code/modules/mob/living/living_verbs.dm

```diff
@@ -263,7 +263,7 @@
 	if(incapacitated(TRUE))
 		return
 
-	if(!resting)
+	if(!resting && !is_ventcrawling)
```

Comment:
Since there's an `else`, I was thinking about this instead:
```
if(!resting)
	if(!is_ventcrawling)
		set_resting(TRUE, FALSE)
else if(action_busy)
```